soccer/StripChart.cpp

```diff
@@ -47,6 +49,30 @@ void StripChart::function(Chart::Function* function) {
     }
 }
 
+void StripChart::xport() {
+    float newMin = _minValue;
+    float newMax = _maxValue;
+    for (unsigned int x = 0; x < _functions.size(); x++) {
+        auto function = _functions[x];
+        
+        std::ofstream outfile("chartData.csv");
+        outfile << "Time,Value\n";
+
+        for (unsigned int i = 0; i < _history->size(); ++i) {
+            float v = 0;
+            if (_history->at(i) && function->value(*_history->at(i).get(), v)) {
+                if (autoRange) {
+                    newMin = min(newMin, v);
+                    newMax = max(newMax, v);
+                }
+                QPointF pt = dataPoint(i, v);
```

Comment:
The segment just above this seems like we're re-creating the exact values that are printed to the stripchart. This has the limitations of (maybe) being inaccurate, and being limited to the size of the stripchart currently. The CSV import really is only useful if it captures all of the data, so it would probably be a better idea to grab this data from the logs. 

I _think_ this can be done by simply not using the datapoint function above, and instead using `(framenumber, v)` as your point. We would need to be careful about this though, since we would probably want to convert `framenumber` into seconds if possible. I think there's a way to get that from the frame somehow.